cnf-tests/testsuites/e2esuite/fec/fec.go

```diff
@@ -66,32 +67,49 @@ var _ = Describe("[fec]", func() {
 	})
 })
 
-func getAcc100Device() (string, string, error) {
+func getAcc100Device() (string, string, bool, error) {
 	nodesWithAcc100, err := getSriovFecNodes()
 	nn, err := nodes.MatchingOptionalSelectorByName(nodesWithAcc100)
 	if err != nil {
-		return "", "", err
+		return "", "", false, err
 	}
 
 	if len(nn) == 0 {
-		return "", "", fmt.Errorf("0 nodes with ACC100 accelerator found")
+		return "", "", false, fmt.Errorf("0 nodes with ACC100 accelerator found")
 	}
 
 	pci, err := getAcc100PciFromNode(nn[0])
 	if err != nil {
-		return "", "", err
+		return "", "", false, err
+	}
+
+	// Get secure boot status from node.
+	acc100Node, err := client.Client.Nodes().Get(context.TODO(), nn[0], metav1.GetOptions{})
```

Comment:
please, consider move this code block to a `isSecureBoot(nodeName string) (bool, error)` function and call it where it's needed.

Reply:
Updated, please take a look one more time.